client/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchOverviewReviews } from './actions/reviewActions';
+import WhoIsStaying from './components/WhoIsStaying/WhoIsStaying';
+import ReviewsAndRatings from './components/ReviewsAndRating/ReviewsAndRatings';
+import LatestReviews from './components/LatestReviews/LatestReviews';
+import Sidebar from './components/Sidebar/Sidebar';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    this.props.fetchOverviewReviews('5b16725b923e9e25353c330d');
```

Comment:
Will everyone have access to your database? Regardless, in the rare chance your db is dropped and you have to re-create, you may want to grab a default review by findOne() instead of findById. Unless, you hard coded the ids, the id will be a changing factor.